### PR TITLE
Add storage and system card service for writing results away 

### PR DESCRIPTION
**Pull request by laurensWe**
# Description

Two services are added, one Storage service which handles writing away of .yaml files to different storage locations the second service is the basic implementation of a system card to be extended for full system card functionality.

[Link all GitHub issues fixed by this PR.](https://github.com/orgs/MinBZK/projects/7/views/3?pane=issue&itemId=63118625)

Resolves #

## Checklist

Please check all the boxes that apply to this pull request using "x":

- [x]  I have tested the changes locally and verified that they work as expected.
- [x]  I have followed the project's coding conventions and style guidelines.
- [x]  I have rebased my branch onto the latest commit of the main branch.
- [x]  I have squashed or reorganized my commits into logical units.
- [x]  I have read, understood and agree to the [Developer Certificate of Origin](../blob/main/DCO.md), which this project utilizes.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -32,6 +32,8 @@ psycopg2-binary = "^2.9.9"
 uvicorn = {extras = ["standard"], version = "^0.29.0"}
 playwright = "^1.44.0"
 pytest-playwright = "^0.5.0"
+pyyaml = "^6.0.1"
+isort = "^5.13.2"
```
Comment (anneschuth) [inline pyproject.toml:36]: What is isort needed for (ruff does this for us, right?)

Comment (laurensWe) [reply pyproject.toml:36]: yes that is right, Christopher called me back on this, but then i didn't remove it will do

**Comment by uittenbroekrobbert** [comment]
I first rebased this branch (locally) onto the setup-mvp-cards branch, as that is ready to merge into main. Due to those changes, I believe some tests don't work anymore. I'll look into this.

File: tests/services/test_storage.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import shutil
+from pathlib import Path
+
+import pytest
+from tad.services.storage import FileSystemWriteService
+from yaml import safe_load
+
+
+@pytest.mark.skip(reason="This is an initialisation function for the tests")
```
[the remark this answers is not in the document]
Comment (ChristopherSpelt) [reply tests/services/test_storage.py]: Changed this to use fixtures

**Comment by laurensWe** [comment]
Closed because it is duplicate of https://github.com/MinBZK/tad/pull/30

**Pull request closed without merging**